ADOPTERS.md

```diff
@@ -27,62 +27,63 @@ updating this list.
+| [Veset](https://veset.tv/) | [@pblgomez](https://github.com/pblgomez) | At Veset we are deploying all our backend resources in production environments to be managed by Crossplane.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| [Hyland Software](https://www.hyland.com/) | [@sethfduke](mailto:seth.duke@hyland.com) | Hyland is utilizing Crossplane to simplify the infrastructure provisioning process for internal development teams, providing a simple means of creating resources, while maintaining control over compliance, security, and best-practices of these resources through a suite of Compositions.                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| [Skillsoft](https://www.skillsoft.com/) | [@brandon-powers](https://github.com/brandon-powers) | At Skillsoft, Crossplane provisions & manages the AWS infrastructure (S3, Athena, and Glue) required by core Apache Kafka services we maintain to power our online learning platform, [Percipio](https://www.skillsoft.com/meet-skillsoft-percipio).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
> we've recently been using Crossplane in production

This is awesome! could you update this description with this same production usage detail, so folks will see that when they look at this list later on 😉 

something like:

```
to power our online learning platform, [Percipio](https://www.skillsoft.com/meet-skillsoft-percipio), in production environments.
```

Reply:
Good call, I've added the above suggestion & tweaked the wording a bit.